lib/releases.js

```diff
@@ -30,11 +30,17 @@ class Release {
 async function fetchReleases () {
   const url = 'https://api.github.com/repos/opentofu/opentofu/releases';
 
+  const headers = {
+    Accept: 'application/vnd.github+json',
+    'X-GitHub-Api-Version': '2022-11-28'
+  };
+
+  if (process.env.GITHUB_TOKEN !== '') {
```

Comment:
empty string is falsy anyway here, I feel like this would work just fine and catch other falsy cases too.
```suggestion
  if (process.env.GITHUB_TOKEN) {
```